rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -404,17 +423,18 @@ def __getitem__(self, field_name):
 			# This shouldn't happen
 			raise Error("Unexpected connector_binding.getAnyValueFromSamples result")
 
-	def get_dictionary(self):
+	def get_dictionary(self, member_name = None):
 		"""Gets a dictionary with the values of all the fields of this sample
 
 		The dictionary keys are the field names and the dictionary values correspond
 		to each field value. To see how nested types, sequences, and arrays are
 		represented, see :ref:`Accessing the data`.
 
-		:return: A dictionary containing all the fields of the sample.
+		:param str element_name: (Optional) The complex member within the type to obtain.
+		:return: A dictionary containing all the fields of the sample, or the complex type if element_name is supplied, or the field of a nested member, which can be another complex type, a sequence or an array.
```

Comment:
I think you need to rewrite and simplify this sentence.

A dictionary containing all the fields of this sample, or if a member_name is specified, all the fields or elements of that member.